packages/node_modules/brightspace-auth-validation/src/index.js

```diff
@@ -38,7 +37,7 @@ function processJwks(jwks, knownPublicKeys, maxKeyAge, skew) {
 
 		const key = knownPublicKeys.has(jwk.kid)
 			? knownPublicKeys.get(jwk.kid).key
-			: createPublicKey ? createPublicKey(jwkToPem(jwk)) : jwkToPem(jwk);
```

Comment:
I think I'd rather keep this one. It doesn't really fall into any serialization problems since the objects are internal, and it's more performant, so if a service doesn't update (since some may be validation-only) I'd like them to at least get the boost here.

Reply:
Done